appeals/web/src/server/lib/mappers/appeal.mapper.js

```diff
@@ -1152,100 +1152,226 @@ export async function initialiseAndMapAppealData(
 		}
 	};
 
-	const appealHasAppellantCostsDocuments = appealDetails?.costs?.appellantFolder?.documents?.filter(
-		(document) => document.latestDocumentVersion?.isDeleted === false
-	).length;
-
 	/** @type {Instructions} */
-	mappedData.appeal.costsAppellant = {
-		id: 'costs-appellant',
+	mappedData.appeal.appealDecision = {
+		id: 'appeal-decision',
 		display: {
 			tableItem: [
 				{
-					text: 'Costs (appellant)',
-					classes: 'appeal-costs-appellant-documentation'
+					text: 'Appeal decision',
+					classes: 'appeal-decision-documentation'
 				},
 				{
-					text: appealHasAppellantCostsDocuments ? 'Received' : '',
-					classes: 'appeal-costs-appellant-status'
+					text: appealDetails.appealStatus === 'complete' ? 'Sent' : 'Awaiting decision',
```

Comment:
Status should come from constants:
`import { APPEAL_CASE_STATUS } from 'pins-data-model';`

Reply:
Well spotted!

Reply:
Done